Pipfile

```
@@ -15,7 +15,7 @@ isort = "*"
 pre-commit = "*"
 pytest = "*"
 vcrpy = ">=4.3.0,!=4.3.1,<4.4.0"  # v4.3.1 broke decode_compressed_response
-urllib3 = "<2" # pin until https://github.com/kevin1024/vcrpy/issues/688 is fixed
+urllib3 = "1.26.16" # pin until https://github.com/kevin1024/vcrpy/issues/688 is fixed
```

To pin the dependency you need to change this to:
```suggestion
urllib3 = "==1.26.16" # pin until https://github.com/kevin1024/vcrpy/issues/688 is fixed
```

But I think we can avoid doing so by updating vcrpy instead.